package.json

```diff
@@ -33,6 +33,8 @@
     "@angular/router": "~6.1.0",
     "core-js": "^2.5.4",
     "nativescript-angular": "~6.1.0",
+    "nativescript-orientation": "^2.2.1",
+    "nativescript-platform-css": "^1.6.6",
```

Comment:
Do we need to check for platform? Or is it because that's the only way to get the screen size?

Reply:
Yes with this plugin it is, we will make our own superduper plugin or possible just make a change request to that existing plugin.